imgui-SFML.cpp

```diff
@@ -1164,7 +1168,7 @@ void RenderDrawLists(ImDrawData* draw_data) {
     }
 
     ImGuiIO& io = ImGui::GetIO();
-    assert(io.Fonts->TexID != (ImTextureID)NULL); // You forgot to create and set font texture
+    assert(io.Fonts->TexID != (ImTextureID) nullptr); // You forgot to create and set font texture
```

Comment:
Do we even need to do that cast to `ImTextureID`? Maybe that's out of scope of this PR though.

Reply:
I think this cast was needed to calm down some other compiler warnings :)